tvb_framework/tvb/core/services/settings_service.py

```diff
@@ -184,7 +184,7 @@ def save_settings(self, **data):
 
         # Storage changed but DB didn't, just copy TVB storage to new one.
         if storage_changed and not db_changed:
-            shutil.copytree(previous_storage, new_storage)
+            shutil.copytree(previous_storage, new_storage, dirs_exist_ok=True)
```

Comment:
Is this line change needed also? or just the one bellow?

Reply:
Yes. shutil.copytree would throw an error because the folder with the db already existed since it was created when the validate db button was pressed on the first startup. dirs_exist_ok=True ensures that the missing filles and dirs are added alongside the db.